### ISSUE TITLE
[Feature request] Inform the guest application that the disc has been ejected if the file path becomes inaccessible.

### ISSUE THREAD

**Issue by Gael2433** [RPCS3/rpcs3#13162]
### Quick summary

Trigger whatever the ps3 does to inform the currently running disc game that the disc has been ejected if the file path for it becomes inaccessible.

### Details

I noticed this while messing around with an already decrypted disc dump burned onto a blank dvd-r, ejecting the disc while the game is running will immediately crash the game with "Verification failed"(see attached log):
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/10338210/RPCS3.log)

I believe that this feature is necessary for accuracy, future disc image support or even booting off original discs someday.

This feature would avoid a crash if an external drive the game dump is on or a network drive is having connection issues or like in my case as described above.

I also noticed that the "Eject Disc" option under "Emulation" is greyed out, I am assuming that this is only implemented for PSOne Classics at this time?

**Comment by Megamouse** [comment]
The eject disc function is only used in very specific games and is not a universal feature.

**Comment by Megamouse** [comment]
On real hardware the game should just be killed usually, which is basically what happens when it crashes anyway.

**Comment by Gael2433** [comment]
Correct, but I believe that he emulation shouldn't crash and instead send the shutdown command to the game in such a situation or pause emulation so it might be possible to continue.

**Comment by elad335** [comment]
There is not much we can do other than force kill the application before it possibly corrupts its own data, although that would be a rare situation I believe.
